### ISSUE TITLE
BUG: Surface pen hardware eraser support

### ISSUE THREAD

**Issue by ascaletty** [zsviczian/obsidian-excalidraw-plugin#1335]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.4.13
	Installer version: v1.4.11
	Operating system: #1 SMP PREEMPT_DYNAMIC Fri Aug 25 22:33:48 UTC 2023 6.4.12-surface
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Excalidraw v1.9.19

**Describe the bug**S
Surface Pen hardware simply does not work, even with the hardware eraser script.

**To Reproduce**
Any time using surface eraser, flipping the pen around does not switch the pen to eraser.

**Comment by traptw1thin** [comment]
Did you run the script after installing it? It didn't work for me until I ran the script:

- Ctrl+P
- search for `excalidraw:`  and find the hardware eraser script
- restart obsidian might be needed

**Comment by ascaletty** [comment]
Yes I did run and restart obsidian, still no luck. My eraser works perfect on xournal, but no luck on obsidian.

**Comment by Jacksery** [comment]
I'm having this issue on Linux using the linux-surface kernel so might be that it translates the eraser end of the pen in a way the plugin doesn't support?

**Comment by zsviczian** [comment]
Consolidating pen / stylus / handwriting / drawing tablet support discussion into #2773.

Closing this as a duplicate so related requests stay centralized. As noted in #2773, hardware-specific pen features and behavior are now tracked there.
